src/packs/abilities/Troubadour_MbinHVcKZQ9kJlOA/6th_Level_Z6jujQ6Szg8O5J8b/Duelist_p9OWPuwuCTBLpK0u/ability_Blood_on_the_Stage_6mkwcnWdj9H170iy.json

```
@@ -0,0 +1,316 @@
+{
+  "folder": "p9OWPuwuCTBLpK0u",
+  "name": "Blood on the Stage",
+  "type": "ability",
+  "_id": "6mkwcnWdj9H170iy",
+  "img": "icons/skills/melee/blade-tip-chipped-blood-red.webp",
+  "system": {
+    "source": {
+      "book": "Heroes",
+      "page": "212",
+      "license": "Draw Steel Creator License"
+    },
+    "_dsid": "blood-on-the-stage",
+    "story": "It’s love and blood or drama and blood. Either way, there’s always blood.",
+    "keywords": [
+      "melee",
+      "strike",
+      "weapon"
+    ],
+    "type": "main",
+    "category": "",
+    "resource": null,
+    "trigger": "",
+    "distance": {
+      "type": "melee",
+      "primary": 1
+    },
+    "damageDisplay": "melee",
+    "target": {
+      "type": "creatureObject",
+      "custom": "",
+      "value": 1
+    },
+    "power": {
+      "roll": {
+        "formula": "@chr",
+        "characteristics": [
+          "agility"
+        ]
+      },
+      "effects": {
+        "OrDSEr7vKXRpCWe6": {
+          "name": "",
+          "img": null,
+          "type": "damage",
+          "_id": "OrDSEr7vKXRpCWe6",
+          "damage": {
+            "tier1": {
+              "value": "12 + @chr"
+            },
+            "tier2": {
+              "value": "18 + @chr"
+            },
+            "tier3": {
+              "value": "24 + @chr"
+            }
+          }
+        },
+        "wIi7T7j7C610hFtm": {
+          "name": "Bleeding EoT",
+          "img": null,
+          "type": "applied",
+          "_id": "wIi7T7j7C610hFtm",
+          "applied": {
+            "tier1": {
+              "potency": {
+                "characteristic": "none"
+              },
+              "display": "",
+              "effects": {}
+            },
+            "tier2": {
+              "display": "",
+              "potency": {
+                "characteristic": "none"
+              },
+              "effects": {}
+            },
+            "tier3": {
+              "potency": {
+                "characteristic": "none"
```

Since this is the alternative to the save ends

```suggestion
                "characteristic": "might"
```